scripts/vitest.scripts.config.js

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
   root: path.resolve(__dirname),
   test: {
     globals: true,
-    environment: 'node'
+    environment: 'node',
+    testTimeout: 10_000
```

Comment:
@jboix since there is no network call when running the tests, what was the reason for adding a timeout?